### ISSUE TITLE
Automatically offer submitting Bodhi update for single package parallel build

### ISSUE THREAD

**Issue by oturpe** [juhp/fbrnch#35]
In #30, single package parallel builds acquired automatic merge offers through `-m` parameter of `fbrnch parallel`.
The missing piece for complete single package version upgrade is submitting Bodhi updates.
Is it possible to have that capacity in `fbrnch parallel`?
I am looking for similar automation of Bodhi updates as exists in `fbrnch build -F`.

**Comment by juhp** [comment]
I think it is already done in 60e7e34.
Just waiting on me to make the next release still, alas

**Comment by juhp** [comment]
I have released 1.1.2 and built it for Rawhide.

**Comment by oturpe** [comment]
Thank you! I will test 1.1.2 as soon as one of my packages get an update that I can apply to all current Fedora releases.

**Comment by juhp** [comment]
Sure thanks - you can comment here or open a new issue if you still see problems :+1:

**Comment by juhp** [comment]
This is in fbrnch-1.2, released yesterday

**Comment by oturpe** [comment]
Tested with 1.2, works as expected:
Bodhi updates created for all Fedora branches except rawhide.

**Comment by juhp** [comment]
Great
